lib/components/artist/shows/large_image_show.js

```diff
@@ -22,11 +22,14 @@ class LargeImageShow extends React.Component {
   }
 
   render() {
+    const show = this.props.show;
+    const image = show.meta_image;
+    const imageURL = image && image.cropped.url;
```

Comment:
@sarahscott @mennenia Another way to deal with null-ing, this is similar to `foo ?: fallback` in C, but that shorthand of the ternary. See https://repl.it/CWx3